src/webui/abstractwebapplication.cpp

```diff
@@ -234,14 +234,15 @@ QString AbstractWebApplication::generateSid()
 
 void AbstractWebApplication::translateDocument(QString& data)
 {
-    const QRegExp regex("QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR");
+    const QRegExp regex("QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR(\\(CONTEXT=([a-zA-Z_][a-zA-Z0-9_]*)([\\)]))?");
```

Comment:
I think that text easier to read when context part is outside.
May be it will be better like this:
```QBT_TR(Downloaded data)QBT_TR[CONTEXT=ContextName]``` ?